examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
+        if os.path.exists(config_default):
+            self.argparse.set_defaults(config=config_default)
```

Comment:
Please move the whole logic to framework.py, so that other examples can benefit from that.